compiler/rustc_error_codes/src/error_codes.rs

```diff
@@ -621,7 +622,7 @@ E0792: include_str!("./error_codes/E0792.md"),
 //  E0488, // lifetime of variable does not enclose its declaration
 //  E0489, // type/lifetime parameter not in scope here
 //  E0490, // removed: unreachable
-    E0523, // two dependencies have same (crate-name, disambiguator) but different SVH
+//  E0523, // removed: unreachable
```

Comment:
Please remove it from here since it's in the list upward.

Reply:
done